pkg/manifests/manifests.go

```diff
@@ -174,6 +174,19 @@ func (f *Factory) RouterDeployment(cr *ingressv1alpha1.ClusterIngress) (*appsv1.
 
 	deployment.Spec.Template.Spec.Containers[0].Image = f.config.RouterImage
 
+	if cr.Spec.HighAvailability != nil && cr.Spec.HighAvailability.Type == ingressv1alpha1.UserDefinedClusterIngressHA {
+		// Expose ports 80 and 443 on the host to provide endpoints for
+		// the user's HA solution.
+		deployment.Spec.Template.Spec.HostNetwork = true
```

Comment:
We use `hostNetwork` instead of `hostPort` for performance reasons (reduce one more network hop)?